### PR TITLE
Obmin 373 redisign button

### PR DESCRIPTION
**Pull request by Mish-hash**

### CODE REVIEW

File: src/components/button-new/styles.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  align-items: center;
+  border-radius: 63px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  width: 286px;
+  height: 40px;
+
```
Comment (cloudmenson) [inline src/components/button-new/styles.ts:16]: width \ heigth should be hierarchical

**Review by cloudmenson — APPROVED** [review]

File: src/components/button-new/ButtonNew.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+export const GreenButton: Story = {
+  args: {
+    text: 'Add advertisement',
+    plus: true,
+    animated: true,
+    colorType: 'green',
+    disabled: false,
+    styleType: 'default',
+  },
```
Comment (ASGladchenko) [inline src/components/button-new/ButtonNew.stories.tsx:23]: sort

File: src/components/button-new/ButtonNew.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+export const BlueButton: Story = {
+  args: {
+    text: 'View',
+    plus: false,
+    animated: true,
+    colorType: 'blue',
+    disabled: false,
+    styleType: 'default',
+  },
```
Comment (ASGladchenko) [inline src/components/button-new/ButtonNew.stories.tsx:34]: same

File: src/components/button-new/ButtonNew.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+export const OutLineButton: Story = {
+  args: {
+    text: 'Show more',
+    plus: false,
+    animated: false,
+    colorType: 'blue',
+    disabled: false,
+    styleType: 'outline',
+  },
```
Comment (ASGladchenko) [inline src/components/button-new/ButtonNew.stories.tsx:45]: same

File: src/components/button-new/styles.ts [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+        `
+      }
+
+      ${
+        colorType == 'green' &&
+        theme.responsive.isDesktop &&
+        css`
+          width: 290px;
+          height: 50px;
+          font-size: 20px;
```
Comment (ASGladchenko) [inline src/components/button-new/styles.ts:64]: props repeat in 72 line 73 line maybe fixed

**Review by Viktoria-Serkiz — APPROVED** [review]

**Review by Voloshkina — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**